app/models/entities/Drug.scala

```diff
@@ -54,5 +56,27 @@ object Drug {
   implicit val mechanismOfActionImpW = Json.format[models.entities.MechanismsOfAction]
   implicit val indicationRowImpW = Json.format[models.entities.IndicationRow]
   implicit val indicationsImpW = Json.format[models.entities.Indications]
-  implicit val drugImpW = Json.format[models.entities.Drug]
+  implicit val DrugXRefImpF = Json.format[models.entities.DrugReferences]
```

Comment:
@JarrodBaker :100: very well used Json transformers. Would you mind explaining this case as a show and tell for @cmalangone, so she can definitely benefit from it. This type of cases agglutinates a lot of know-how and it is quite positive to share